include/yaramod/utils/utils.h

```diff
@@ -45,7 +46,7 @@ bool startsWith(const std::string& str, const T& withWhat)
  * @return Number converted to string.
```

Comment:
Nitpick: Missing docstring with the new argument

Reply:
Removed parameter